experimental/packages/opentelemetry-instrumentation-fetch/src/fetch.ts

```diff
@@ -301,7 +301,7 @@ export class FetchInstrumentation extends InstrumentationBase<
       ): Promise<Response> {
         const self = this;
         const url = web.parseUrl(
-          args[0] instanceof Request ? args[0].url : args[0]
+          args[0] instanceof Request ? args[0].url : String(args[0])
```

Comment:
For later version for TypeScript the args[0] is identified as a `Request`, `string` or `URL` and the `URL` can't be passed to the `parseUrl` so using the `String()` constructor to cause the URL variant to become a string and keep TypeScript happy.